### ISSUE TITLE
How to present comparison in sensitive distance performance?

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/aframe#439]
Should it be:
- Plot curves over one another
- Ratios
- Percent difference
- Difference in terms of number of number of standard deviations of control curve

Definitely applies to:
- reseeded training
- inference sampling rate

Not sure if applies to:
- hyperparameter settings
- Interval vs. original test set
Do we just plot the SV curves against one another for these?
